src/transaction/atomic_replication_helper.cpp

```diff
@@ -1092,6 +1092,7 @@ namespace cublog
   pgbuf_fix_or_ordered_fix (THREAD_ENTRY *thread_p, VPID vpid, LOG_RCVINDEX rcvindex,
 			    std::unique_ptr<PGBUF_WATCHER> &watcher_uptr, PAGE_PTR &page_ptr)
   {
+    constexpr PAGE_FETCH_MODE fetch_mode = OLD_PAGE_IF_IN_BUFFER_OR_IN_TRANSIT;
```

Comment:
Actually, it is the same value as used in `atomic_replicator::atomic_replicator` ctor

Reply:
Since the fetch mod is a memeber variable of replicator, how about putting the function `pgbuf_fix_or_ordered_fix` into atomic_replicator as a member function?